huaweicloud/services/rds/resource_huaweicloud_rds_mysql_account.go

```diff
@@ -40,182 +44,312 @@ func ResourceRdsAccount() *schema.Resource {
 				Computed: true,
 			},
 			"instance_id": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the RDS Mysql instance.`,
 			},
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the username of the DB account.`,
+			},
+			"hosts": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
+				Description: `Specifies the IP addresses that are allowed to access your DB instance.`,
 			},
 			"password": {
-				Type:      schema.TypeString,
-				Required:  true,
-				Sensitive: true,
+				Type:        schema.TypeString,
```

Comment:
1. **Sensitive** is missing
2. should define the `Required` parameter in head of `Optional` parameters

Reply:
done